### PR TITLE
Randomize the Timeout in Get Tools

### PR DESCRIPTION
**Pull request by thinkocapo**
it's according to a [long-tail](https://en.wikipedia.org/wiki/Long_tail) distribution of values for the timeout

## How
`time.sleep(numpy.random.lognormal(0.75, .6, 1)[0])`

## Result
image
https://sentry.io/organizations/testorg-az/performance/?project=5260888&project=1428657&query=transaction%3Ahttps%3A%2F%2Fwcap-react-m3uuizd7iq-uc.a.run.app%2F&statsPeriod=7d

#### Prod GCP

[js error /checkout ](https://sentry.io/organizations/testorg-az/discover/will-frontend-react:672a80057f2846ba9c72c4b9d9f11eb0/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&project=5260888&project=1428657&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[js tx /checkout](https://sentry.io/organizations/testorg-az/discover/will-frontend-react:cc8b5053050c43638b11f32493f0fdf7/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&project=5260888&project=1428657&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[py tx /checkout](https://sentry.io/organizations/testorg-az/discover/python-test:6c42b4824cc6483a9be57db7d8eaf44e/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&project=5260888&project=1428657&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[py error /checkout](https://sentry.io/organizations/testorg-az/discover/python-test:174693b8358c4f26b7e4f1d94245daa1/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&project=5260888&project=1428657&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[py tx pageload](https://sentry.io/organizations/testorg-az/discover/python-test:db0deba2f5ab48a48fc2663f1b6ee836/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[js tx pageload](https://sentry.io/organizations/testorg-az/discover/will-frontend-react:7e4b337327864ab18f91b3081b74c208/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

#### Dev without Docker
[js tx /checkout](https://sentry.io/organizations/testorg-az/discover/will-frontend-react:ebe5db8f0c9e4d588b6ea4ac2fc450f9/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[py error /checkout](https://sentry.io/organizations/testorg-az/discover/python-test:913237422dd749efb5ab433015cea5d9/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[py tx /checkout](https://sentry.io/organizations/testorg-az/discover/python-test:b110a98da9f640b18fe2938043bdedcf/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[py tx pageload](https://sentry.io/organizations/testorg-az/discover/python-test:a3e4e27a105e4c6c96cf1888ef956851/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

[js tx /pageload](https://sentry.io/organizations/testorg-az/discover/will-frontend-react:bc6befe5beb44c759d2f7f4a8903e15a/?field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&widths=-1&widths=-1&widths=-1&widths=-1&widths=-1)

### CODE REVIEW

File: flask/app.py [hunk 1]
```diff
@@ -2,22 +2,24 @@
...
-KEY = DSN.split('@')[0]
-try:
-    if KEY.index('s') == 4: # http vs https
-        KEY = KEY[:4] + KEY[5:]
-except Exception as err:
-    print('DSN key w/ http from self-hosted')
-PROXY = 'localhost:3001'
-MODIFIED_DSN_FORWARD = KEY + '@' + PROXY + '/2'
-MODIFIED_DSN_SAVE = KEY + '@' + PROXY + '/3'
+# use this if sending test data to a proxy and not Sentry
```
Comment (thinkocapo) [inline flask/app.py:10]: this block of code is used when sending 100 - 1,000 events in a row to a proxy for saving in an offline database, and replaying the events later.

File: react/src/index.js [hunk 2]
```diff
@@ -12,15 +12,20 @@ import { createStore, applyMiddleware } from 'redux'
 import logger from 'redux-logger'
 import rootReducer from './reducers'
 
-// these modified DSN's are for working with test data. You can ignore them.
-let DSN = process.env.REACT_APP_DSN
-let KEY = DSN.split('@')[0]
-if (KEY.indexOf('s') === 4) {
-  KEY = KEY.replace('s', '') // http vs https
+/* use this if sending test data to a proxy and not Sentry
```
Comment (thinkocapo) [inline react/src/index.js:15]: this block of code is used when sending 100 - 1,000 events in a row to a proxy for saving in an offline database, and replaying the events later.

Comment (ndmanvar) [reply react/src/index.js:15]: 👍

File: flask/app.py (hunk 1 above)
Comment (ndmanvar) [reply flask/app.py:10]: 👍

**Pull request merged**